### ISSUE TITLE
pimatic max, only set temperature by rule if mode is set to auto 

### ISSUE THREAD

**Issue by michaelberg79** [pimatic/pimatic#779]
Hello,
Currently, i using following rules for set the heating by rule:

if its after 00:00 and before 08:30 and [its sunday or its saturday] and $bhbad.mode = "auto" then set temp of BM Bad to 17
if its after 00:00 and before 08:30 and [its sunday or its saturday] and $bhwohnleft.mode = "auto" then set temp of BM Bad to 17

It would be much easier if pimatic dont changes the temperature by rule if the mode is set to manuell. In my mention this should be the regular behavior. Now I need for each time and heating one rule and cant conclude they.

How i can set the label enhancement?

best regards 

Michael Berg

**Comment by mwittig** [comment]
Thanks for your report. I agree it'll make writing rules a easier if the set-action is only performed in auto mode. May be this can even be regarded as common behaviour of the "HeatingThermostat" device class. 

Note, it may take a couple of weeks before someone gets assigned to this ticket.

**Comment by leader21** [comment]
agreed! rules shouldn't change the behavior of the thermostats when switched in manual mode!
actually i am using the internal max timetable for having a basic heating program. therefore i have not faced this issue before.
maybe @georg90 is able to have a look into that?

**Comment by Icesory** [comment]
Weeks? I will look into this in the next days xD

**Comment by mwittig** [comment]
That would be nice. However, do you have access permissions? If not we may not be able to merge the pull requests.

**Comment by Icesory** [comment]
No i have only permissions for all the homeduino related stuff.

I have not testet it but eventually this solves the problem.
https://github.com/Icesory/pimatic/commit/84999d0adfdb16b1321755995dbda3aa53d09f2a
Can some one load this file into his pimatic an test it?

**Comment by n3roGit** [comment]
i use this rule to have all sensors on manu:

if $thermostat1.mode is "auto" or $thermostat2.mode is "auto" or $thermostat3.mode is "auto" or $thermostat4.mode is "auto" or $thermostat5.mode is "auto" then set mode of thermostat1 to "manu" and set mode of thermostat2 to "manu" and set mode of thermostat3 to "manu" and set mode of thermostat4 to "manu" and set mode of thermostat5 to "manu"

the logic to controll it is only in pimatic. so auto is not needed :)
